benchmarks/cpp/nvfuser/scale_bias_relu.cpp

```diff
@@ -49,7 +49,7 @@ static void setupSBR(Fusion* fusion, DataType dtype) {
     bias = castOp(DataType::Float, bias);
   }
 
-  auto scale_bias = add(mul(x, scale), bias);
+  auto scale_bias = sub(mul(x, scale), bias);
```

Comment:
Did you need to do this to get stuff to compile? The test will probably fail massively, but hopefully you don't need to make any changes to `at::add()` calls anywhere.